# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/transforms/big-axis-aligned-scale.yaml
# Commit: 9fa6f71111d0
# Full Hash: 9fa6f71111d0d77477abc4502b721d8d4ba23604
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2022-02-20 21:58:21
# Regressor Bug: 1749380
# File Overlap Count: 1
# Description:
#   Bug 1749380 - Part 1 - Improve how WR handles bounding rects for off-screen surfaces r=gfx-reviewers
#   
#   This patch introduces a number of subtle but important changes
#   to how we deal with off-screen surfaces. The overall goals are:
#   
# ==============================================================================

diff -r 336a2aa0a29e -r 9fa6f71111d0 gfx/wr/wrench/reftests/transforms/big-axis-aligned-scale.yaml
--- a/gfx/wr/wrench/reftests/transforms/big-axis-aligned-scale.yaml	Sat Feb 19 23:27:02 2022 +0000
+++ b/gfx/wr/wrench/reftests/transforms/big-axis-aligned-scale.yaml	Sun Feb 20 18:56:54 2022 +0000
@@ -3,7 +3,7 @@
   items:
     - type: stacking-context
       transform-style: preserve-3d
-      transform: scale(-2, 44, 44727)
+      transform: scale(-2, 44, 1)
       items:
         - type: rect
           bounds: [ -100, -100, 200, 101 ]